rules/windows/process_creation/proc_creation_win_susp_service_dacl_modification_set_service.yml

```diff
@@ -0,0 +1,40 @@
+title: Suspicious Service DACL Modification Via Set-Service Cmdlet
+id: a95b9b42-1308-4735-a1af-abb1c5e6f5ac
+related:
+    - id: 99cf1e02-00fb-4c0d-8375-563f978dfd37
+      type: derived
+status: experimental
+description: Detects suspicious DACL modifications via the "Set-Service" cmdlet using the "SecurityDescriptorSddl" flag (Only available with PowerShell 7) that can be used to hide services or make them unstopable
+author: Nasreddine Bencherchali
+references:
+    - https://www.sans.org/blog/red-team-tactics-hiding-windows-services/
+    - https://docs.microsoft.com/pt-br/windows/win32/secauthz/sid-strings
+date: 2022/10/18
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\pwsh.exe'
+        - OriginalFileName: 'pwsh.dll'
+    selection_sddl_flag:
+        CommandLine|contains:
+            - '-SecurityDescriptorSddl '
+            - '-sd '
+    selection_set_service:
+        CommandLine|contains|all:
+            - 'Set-Service '
+            - 'D;;'
+        CommandLine|contains:
+            - ';;;IU'
+            - ';;;SU'
+            - ';;;BA'
+            - ';;;SY'
+            - ';;;WD'
+    condition: all of selection_*
+falsepositives:
+    - Unknown
+level: high
+tags:
+    - attack.persistence
+    - attack.t1543.003
```

Comment:
Is this a candidate for a script block logging rule (if that's possible for PS7)?